app/src/main/java/com/example/photofy/activities/MainActivity.java

```diff
@@ -45,5 +57,38 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         });
         // Set default selection
         bottomNavigationView.setSelectedItemId(R.id.action_home);
+
+        btnLogout.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                logout();
+            }
+        });
     }
+
+    private void logout() {
+        ParseUser.logOutInBackground(new LogOutCallback() {
+            @Override
+            public void done(ParseException e) {
+                if (e != null) {
+                    Log.e(TAG, "Issue with logout", e);
+                    return;
```

Comment:
`return` not needed here

Reply:
Also, show error to the user (through toast or taking him to an error activity).  this can be done as part of different pull request.

Reply:
resolved in PR 4